Jenkinsfile

```diff
@@ -129,7 +133,7 @@ String unit_packages() {
                            'pmix numactl-devel ' +
                            'libipmctl-devel ' +
                            'python36-tabulate '
-        if (quickbuild()) {
+        if (need_qb) {
```

Comment:
I'm not seeing the close brace for this `if`.

Reply:
It's there. Maybe collapsed in your view? Try expanding it.

Reply:
Oh wait.  Right.  You are just changing the conditional, not adding a gate.  Apologies.